itests/wdpost_test.go

```diff
@@ -268,6 +274,12 @@ func TestWindowPostBaseFeeBurn(t *testing.T) {
 	client, miner, ens := kit.EnsembleMinimal(t, kit.MockProofs(), opts)
 	ens.InterconnectAll().BeginMining(blocktime)
 
+	// Ideally we'd be a bit more precise here, but getting the information we need from the
+	// test framework is more work than it's worth.
+	//
+	// We just need to wait till all upgrades are done.
+	client.WaitTillChain(ctx, kit.HeightAtLeast(20))
```

Comment:
@Stebalien we could add a predicate that tests against `StateNetworkVersion`? We'd need to expand the signature of `ChainPredicate` so we can pass in the FullNodeAPI.

Reply:
Yeah. That would work. I guess I could also just pass a closure here and capture it.

Reply:
Good idea.